src/internal.c

```diff
@@ -20735,7 +20749,7 @@ int BuildMessage(WOLFSSL* ssl, byte* output, int outSz, const byte* input,
                 ssl->keys.dtls_sequence_number_lo = dtls_sequence_number_lo;
             }
     #endif
-			}
+            }
```

Comment:
Having this at the same indent as the previous brace looks wrong.

Reply:
I agree but I'm only cleaning up hard tabs and making minimal & local changes to fix the test failures.  We can toss it back to the owner of the changes (@julek-wolfssl) to fix it nicer and keep the ownership on all those lines of code matching the person that wrote them.